honeybee/radiance/command/rcollate.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+from commandBase import RadianceCommand
+import os
+import sys
+from ..datatype import *
+from ..datatype import RadInputStringFormatter as Rfmt
+
+
+class Rcollate(RadianceCommand):
+    u"""
+    rcollate - resize or transpose matrix data
+
+    Attributes:
+
```

Comment:
@mostaphaRoudsari The usage thing actually makes a lot of sense for me. In my radiance wrapper I wrote some usage tests a while ago. The issue with some of the radiance binaries is that we at least need some minimal geometry files, test file etc. if we are to test their performance at run-time. Other, probably less time consuming, option would be to just test for the command string outputs. Do you have something implemented already that I could follow ?